pytest_splunk_addon_ui_smartx/components/controls/single_select.py

```diff
@@ -155,7 +155,10 @@ def get_single_value(self):
             self.select(selected_val, open_dropdown=False)
         elif self.searchable:
             self.input.send_keys(Keys.ESCAPE)
-        elif first_element:
+        else:
+            for each in self.get_elements('values'):
+                first_element = each
+                break
             self.select(first_element.text.strip(), open_dropdown=False)
```

Comment:
Can you please update the code to: 
```
else:
     for each in self.get_elements('values'):
             self.select(each.text.strip(), open_dropdown=False)
             break            
```
To handle the scenario if the dropdown did not open and there are no option visible.

Reply:
Done